packages/xo-server-sdn-controller/src/index.js

```diff
@@ -243,11 +302,18 @@ class SDNController extends EventEmitter {
         )
       })
     )
+
+    await Promise.all(
+      map(this._crossPoolNetworks, crossPoolNetwork =>
+        this._electNewPoolCenter(crossPoolNetwork)
+      )
+    )
   }
 
   async unload() {
     this._ovsdbClients = []
     this._poolNetworks = []
+    this._crossPoolNetworks = {}
     this._newHosts = []
```

Comment:
Could we unify this with the constructor in another PR?

Reply:
Good idea, noted! :)